open_discussions/settings.py

```diff
@@ -426,6 +426,11 @@
 
 OPEN_DISCUSSIONS_DEFAULT_SITE_KEY = get_string("OPEN_DISCUSSIONS_DEFAULT_SITE_KEY", None)
 
+OPEN_DISCUSSIONS_UNSUBSCRIBE_TOKEN_MAX_AGE_SECONDS = get_int(
+    'OPEN_DISCUSSIONS_UNSUBSCRIBE_TOKEN_MAX_AGE_SECONDS',
+    60 * 60 * 24 * 5  # 5 days
```

Comment:
should this be 7 days, so the latest weekly email that the user received will always have a valid token?

Reply:
yeah, we can do that.